code/modules/jobs/job_types/legion.dm

```diff
@@ -239,41 +278,41 @@ commented out pending rework*/
 	ADD_TRAIT(H, TRAIT_BIG_LEAGUES, src)
 	ADD_TRAIT(H, TRAIT_IRONFIST, src)
 	ADD_TRAIT(H, TRAIT_HARD_YARDS, src)
+	ADD_TRAIT(H, TRAIT_LIFEGIVER, src)
 
 /datum/outfit/job/CaesarsLegion/Legionnaire/f13decanvet
-	name = "Legion Veteran Decanus"
-	jobtype = /datum/job/CaesarsLegion/Legionnaire/f13decanvet
-	id = /obj/item/card/id/dogtag/legveteran
-	suit = /obj/item/clothing/suit/armor/f13/legion/heavy
-	mask = /obj/item/clothing/mask/bandana/legion/legdecan
-	neck = /obj/item/storage/belt/holster
-	gloves = /obj/item/clothing/gloves/legion/plated
-	glasses = /obj/item/clothing/glasses/sunglasses/big
-	r_pocket = /obj/item/flashlight/lantern
-	box = /obj/item/storage/survivalkit_tribal/chief
-	backpack_contents = list(
-		/obj/item/ammo_box/a357 = 1,
-		/obj/item/gun/ballistic/revolver/colt357 = 1,
-		/obj/item/restraints/handcuffs = 1,
-	/obj/item/storage/bag/money/small/legion = 1,
-	)
+	name =				"Legion Veteran Decanus"
+	jobtype =			/datum/job/CaesarsLegion/Legionnaire/f13decanvet
+	id =				/obj/item/card/id/dogtag/legveteran
+	suit =				/obj/item/clothing/suit/armor/f13/legion/heavy
+	mask =				/obj/item/clothing/mask/bandana/legion/legdecan
+	neck =				/obj/item/storage/belt/holster
+	gloves =			/obj/item/clothing/gloves/legion/plated
+	glasses =			/obj/item/clothing/glasses/sunglasses/big
+	r_pocket =			/obj/item/flashlight/lantern
+	box =				/obj/item/storage/survivalkit_tribal/chief
+	backpack_contents = list(
+						/obj/item/ammo_box/a357 = 1,
+						/obj/item/gun/ballistic/revolver/colt357 = 1,
+						/obj/item/restraints/handcuffs = 1,
+						/obj/item/storage/bag/money/small/legion = 1,	)
 
 /datum/outfit/loadout/decvetfront
 	name =			"Lead from the front"
 	head = 			/obj/item/clothing/head/helmet/f13/legion/heavy
 	suit_store =	/obj/item/twohanded/fireaxe
 	backpack_contents = list(
 					/obj/item/gun/ballistic/automatic/smg/cg45=1,
-					/obj/item/ammo_box/magazine/cg45=2)
+					/obj/item/ammo_box/magazine/cg45=2,
+					/obj/item/grenade/smokebomb = 1,	)
 
 /datum/outfit/loadout/decvetrear
-	name = "Lead from the rear"
-	head = /obj/item/clothing/head/helmet/f13/legion/vet/decan
-	suit_store = /obj/item/gun/ballistic/automatic/m1garand/sks
+	name =			"Lead from the rear"
+	head =			/obj/item/clothing/head/helmet/f13/legion/vet/decan
+	suit_store =	/obj/item/gun/ballistic/automatic/m1garand/sks
```

Comment:
Please don't do this.
On tg this kind of pattern was regexed away, and we want to follow their style as much as possible, for compatibility.
Plus, this pattern won't be the same on different editors, and can cause confusion.